### PR TITLE
adding action drop down on user management

### PR DESCRIPTION
**Pull request by VineetBala-AOT**
*Description of changes:*
- Added new type for membership type
- Changes to add action drop down on user management
- Removed the add user button

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-api/migrations/versions/779d9125c8cb_add_reviewer_type.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+down_revision = '8def759e43d9'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('membership', 'type', type_=sa.Text())
+    op.execute('DROP TYPE membershiptype;')
+    op.execute('CREATE TYPE membershiptype AS ENUM (\'TEAM_MEMBER\', \'REVIEWER\');')
```
Comment (jadmsaadaot) [inline met-api/migrations/versions/779d9125c8cb_add_reviewer_type.py:23]: do we have to drop, can't we just alter

File: met-api/migrations/versions/779d9125c8cb_add_reviewer_type.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    op.alter_column('membership', 'type', type_=sa.Text())
+    op.execute('DROP TYPE membershiptype;')
+    op.execute('CREATE TYPE membershiptype AS ENUM (\'TEAM_MEMBER\', \'REVIEWER\');')  
+    op.alter_column('membership', 'type', type_=sa.Enum('TEAM_MEMBER', 'REVIEWER', name='membershiptype'), postgresql_using='type::membershiptype')
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('membership', 'type',
```
Comment (jadmsaadaot) [inline met-api/migrations/versions/779d9125c8cb_add_reviewer_type.py:30]: how is the membershiptype downgraded here?

File: met-web/src/components/userManagement/listing/ActionsDropDown.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+    const ITEMS: ActionDropDownItem[] = useMemo(
+        () => [
+            {
+                value: 1,
+                label: 'Assign Role',
+                action: () => {
+                    {
+                        setUser(selectedUser);
+                        assignRoleModalOpen ? <AssignRoleModal /> : null;
```
Comment (jadmsaadaot) [inline met-web/src/components/userManagement/listing/ActionsDropDown.tsx:50]: why do we need handleChange. Can we not just assign the appropriate modal open here? Also we are mixing javascript and JSX here. I think this  action should just perform an action and not add any jsx

**Review by jadmsaadaot — CHANGES_REQUESTED** [review]
Refactor ActionsDropDown please

File: met-api/migrations/versions/779d9125c8cb_add_reviewer_type.py (hunk 1 above)
Comment (VineetBala-AOT) [reply met-api/migrations/versions/779d9125c8cb_add_reviewer_type.py]: Thanks, changed it to be alter

File: met-api/migrations/versions/779d9125c8cb_add_reviewer_type.py (hunk 2 above)
Comment (VineetBala-AOT) [reply met-api/migrations/versions/779d9125c8cb_add_reviewer_type.py:29]: Just downgrading the column, keeping the existing type.

File: met-web/src/components/userManagement/listing/ActionsDropDown.tsx (hunk 3 above)
Comment (VineetBala-AOT) [reply met-web/src/components/userManagement/listing/ActionsDropDown.tsx]: removed handleChange and made changes to assign the appropriate modal open, thanks

**Review by djnunez-aot — APPROVED** [review]

**Pull request merged**